aks/extra-variables.tf

```diff
@@ -1,14 +1,16 @@
-variable "node_resource_group_name" {
-  description = "The Resource Group of the Managed Kubernetes Cluster."
-  type        = string
-}
-
 variable "metrics_storage" {
   description = "Azure Blob Storage configuration values for the storage container where the archived metrics will be stored."
```

Comment:
If you eventually change the description on the Loki or Thanos module, I think they should be the same here.